### ISSUE TITLE
Several bugs

### ISSUE THREAD

**Issue by SeBL4RD** [MaxLastBreath/TOTK-mods#43]
Deep underground:

-Miasm bug completely, when you step on it, there's a big delay. Sometimes walking outside the miasm is considered as being inside it. 
(Miasm bug happen only higher than 30 fps)

-Starting a fight or throwing Lumos seeds causes the game to stutter every second, making it unplayable.

(1.21)
RTX 2070 Super, Ryzen 7 3700x, 32 Gigs, 970 Evo Plus
![Screenshot_253](https://github.com/MaxLastBreath/TOTK-mods/assets/45276149/62e65932-b84b-47b4-9407-5213700e0802)
[yuzu_log.txt](https://github.com/MaxLastBreath/TOTK-mods/files/13464590/yuzu_log.txt)

**Comment by SeBL4RD** [comment]
Miasm bug : 
https://youtu.be/qn_-NpYVv-g

**Comment by MaxLastBreath** [comment]
this should be reported to yuzu and not here? I don't see an issue.

**Comment by SeBL4RD** [comment]
> this should be reported to yuzu and not here? I don't see an issue.


Are you serious?
Just play the game, try without mods, or with only 30 FPS with yours. Lets see what happen.
I wouldn't come here if it were related to Yuzu. I've just tried your mods, since I'm in 1.2.1. 
Since I switched to 1.2.1, (when the patch was released), I had no more mods installed, I've only just tested yours, and I reproduce the bug very easily.
